extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/view.html

```diff
@@ -0,0 +1,32 @@
+{#include main}
+{#title}Static Resources{/title}
+{#style}
+{/style}
+{#body}
+ {#let folder=currentRequest.params.get("path")}
+  <div class="row">
+    <div class="col">
+      {folder}
+    </div>
+  </div>
+  {#for childOfFolder in info:staticResourceInfo.resourceMap.get(folder).children}
+    <div class="row">
+      <div class="col-1"></div>
+      <div class="col">
+        {#if !childOfFolder.isFolder}
+          <i class="fas fa-file-alt" style="margin-right:5px; margin-top:5px"></i>
+          <a href="/{childOfFolder.lastName}">
+            {childOfFolder.lastName}
+          </a>
+        {#else}
+          <i class="far fa-folder-open" style="margin-right:5px; margin-top:5px"></i>
+          <a href="view?path=/{childOfFolder.lastName}">
```

Comment:
Same, here you're missing `view?path={folder}/{childOfFolder.lastName}`

Reply:
Done :)